bindings/rust/aws-lc-sys-template/README.md

```diff
@@ -0,0 +1,40 @@
+# aws-lc-sys
+
+[![crates.io](https://img.shields.io/crates/v/aws-lc-sys.svg)](https://crates.io/crates/aws-lc-sys)
+
+Low level AWS-LC bindings for the Rust programming language. We do not recommend directly relying on these bindings.
+
+[Documentation](https://github.com/awslabs/aws-lc).
+
+## Release Support
+
+This crate pulls in the source code of AWS-LC to build with it. Bindings for platforms we officially support are pre-generated. To generate bindings for any platforms where pre-generated bindings aren't available, you can use the `generate_bindings` feature to do so.
+
+### Pregenerated Bindings Availability
+
+CPU|OS
+-------------|-------------
+x86|Ubuntu
+x86-64|Ubuntu
+arm-64|Ubuntu
+x86-64|MacOS
```

Comment:
Same comment as above -- the pregenerated bindings will be used on any Linux host (not just Ubuntu).